gpustat/core.py

```diff
@@ -425,7 +425,7 @@ def __repr__(self):
 
     def print_formatted(self, fp=sys.stdout, force_color=False, no_color=False,
                         show_cmd=False, show_user=False, show_pid=False,
-                        show_power=None, show_fan_speed=None, gpuname_width=16,
+                        show_power=None, show_fan_speed=None, show_all=False, gpuname_width=16,
```

Comment:
If we pop out `show_all` from args from main, we won't need this parameter here.

Reply:
Hi, thank you for your reply. I resolve this problem by deleting the `show_all` attribute from args in main